august/data.py

```diff
@@ -119,3 +119,13 @@
         ]
 
 }
+# 'clear', 'copy', 'fromkeys', 'get', 'items', 'keys', 'pop', 'popitem', 'setdefault', 'update', 'values']
+def student_search():
+    uname = input("Please enter student name: ")
+    for x in data["students"]:
+        if x["name"] == uname:
+            print("Student in the group!")
+            exit(1)
+    print("Not found")
```

Comment:
we want to assign student to the student dictionary and return the found student dictionary

Reply:
Made changes, ready for review